static/app/views/issueDetails/actions/publishModal.tsx

```diff
@@ -51,7 +51,7 @@ export default function PublishIssueModal({
   const isPublished = group?.isPublic;
   const hasStreamlinedUI = useHasStreamlinedUI();
   const handleShare = useCallback(
-    (e: React.MouseEvent<HTMLButtonElement> | null, reshare?: boolean) => {
+    (e: React.ChangeEvent<HTMLInputElement> | null, reshare?: boolean) => {
```

Comment:
I know this is probably out of scope but I absolutely don’t understand why we have this wrapped in `useCallback`. It has two usages, one in an inline function and one where we pass it to `Switch` directly. None of this needs memoization, it makes the code more verbose and it has an `onToggle` prop in the dependency array, where the only usage is an inline function again, so the memoization won’t even work - it will be re-memoized on every render 🤷